src/main/frontend/extensions/pdf/pdf.css

```
@@ -56,15 +56,22 @@ input::-webkit-inner-spin-button {
     position: relative;
   }
 
-  &-toolbar {
+&-header {
+    display: flex;
+    justify-content: flex-end;
     position: absolute;
     top: 0;
-    right: 20px;
-    width: 90%;
-    height: 42px;
+    right: 0px;
+    width: 100%;
```

```suggestion
    right: 10px; // Allows clicking the scrollbar
    width: calc(100% - 10px);
```
The right property fixes https://github.com/logseq/logseq/issues/4509. The suggestion improves the previous implementation a little bit.